aries-site/src/pages/whats-new.mdx

```diff
@@ -53,6 +53,34 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## March 15-19, 2021
+
+### Created Pagination page
+
+- Added guidance and usage examples for when to use [pagination](https://design-system.hpe.design/components/pagination).
+
+### Created Spinner page
+
+- Spinner Page was added onto the site. You can find more about the spinner [here](https://design-system.hpe.design/components/spinner)
+
+### Resolve X-Frame-Options
+
+- Resolved one of the security issues that were found. Read more about the issue [here](https://github.com/grommet/hpe-design-system/issues/1466) 
+
+## March 8-9, 2021
+
+### Conducted a design-sprint to discuss Page-Layouts 
+
+## March 1-5, 2021
+
+### Replaced the light/dark mode button.
```

Comment:
We can remove this section. It doesn't really affect consumers of the Design System.